vars/Debian.yml

```diff
@@ -0,0 +1,3 @@
+---
+
+elasticsearch_package: elasticsearch=7.13.2
```

Comment:
I would still rather not default to installing any particular version — just `elasticsearch`. Can you drop the new `vars/*` files, and the `Include OS-specific variables.` task?

Reply:
i have dropped them now :)